cmd/common_helpers.go

```diff
@@ -99,6 +99,14 @@ func getNamespaceID(id string) (int, error) {
 }
 
 func printGroupsOut(cmd *cobra.Command, groups ...*gitlab.Group) {
+	if len(groups) == 0 {
```

Comment:
* This condition should only apply during the `default` case. It is ok to print the default nothing format of YAML and JSON to users.
* Let's also make it simple by just saying _"The get command returned no results."_,  if get command result length returned is zero.

Some other notes..

* Please keep the column length to 80 when typing long code.
* Use `%s` instead of `%v` when formatting a string.